code/modules/events/event.dm

```diff
@@ -91,6 +90,7 @@
   *
   * Allows you to start before announcing or vice versa.
   * Only called once.
+  * Ensure no sleep is called. Use INVOKE_ASYNC to call procs which do.
   */
 /datum/event/proc/start()
 	return
```

Comment:
Put the `SHOULD_NOT_SLEEP(TRUE)` dreamchecker flags in these procs

Reply:
Those will call out false positives. Such as `emote` having a sleep in it when you flip. `emote` being called on `death` which is called on `gib` etc... I couldn't be bothered to fix the false positives